microsoft-365/security/defender-endpoint/linux-preferences.md

```diff
@@ -572,7 +572,7 @@ To verify that your /etc/opt/microsoft/mdatp/managed/mdatp_managed.json is worki
 - automatic_definition_update_enabled
 
 > [!NOTE]
-> For the mdatp_managed.json to take effect, no restart of the `mdatp` deamon is required.
+> For the mdatp_managed.json to take effect, restart of the `mdatp` deamon is required only in cases when `cloud-diagnostic` or `log-rotation-parameters` configs are updated.
```

Comment:
Reword as follows:

No restart of mdatp deamon is required for changes to **most** configs in mdatp_managed.json to take effect.
**Exception**: Following configs will require deamon restart to take effect:
- cloud-diagnostic
- log-rotation-parameters

Reply:
Done